hack/tools/go.mod

```diff
@@ -1,10 +1,12 @@
 module github.com/metal3-io/ip-address-manager/hack/tools
 
-go 1.21
+go 1.22.0
```

Comment:
Code generator 0.30.x bumps go.mod to go 1.22, we need to wait with this.

/hold